tests/backend/notification.c

```diff
@@ -41,6 +41,7 @@ invoke_action (gpointer data)
 static gboolean
 handle_add_notification (XdpDbusImplNotification *object,
                          GDBusMethodInvocation *invocation,
+                         GUnixFDList *fd_list,
```

Comment:
I think this ended up in the wrong commit? Seems like it was supposed to be part of the previous commit